frontend/src/store/player-element.ts

```diff
@@ -118,63 +125,28 @@ class PlayerElementStore extends CommonStore<PlayerElementState> {
     }
   };
 
-  private readonly _setSsaTrack = (trackSrc: string, attachedFonts?: string[]): void => {
+  private _fetchSSATrack = async (trackSrc: string) => {
+    fetch(trackSrc).then((body) => body.text()).then((text) => {
+      this._asssub = new ASSSUB(text, mediaElementRef.value as (HTMLVideoElement), { resampling: 'video_width' });
+    });
+  };
```

Comment:
Use our current axios instance and ``await`` syntax please.
Also, mind adding a comment explaining why the default resampling (video_height) method from the library can't be used but video_width?

Reply:
Understood, I can do that, my apologies, it was a quick fix! As per why I'm not using video_height instead on my silly screen which is an ultrawide monitor video_width actually works better most of the time, but I should've considered other screen sizes.

Reply:
Alright, I believe this is working in the latest commit.